ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java

```diff
@@ -817,10 +819,10 @@ public static int get_permuted_index(int index, int list_size, Bytes32 seed) {
 
       // This needs to be unsigned modulo.
       int pivot =
-          (int)
```

Comment:
Is there a reason to remove toIntExact? it seems safer than (int)

Reply:
This change shows (int) being removed and toIntExact being added :)